src/views/CompareWithAzion/blocks/grade-block.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="rounded surface-border border">
+    <div
+      v-if="props.items.length === 0 || !props.items"
```

Comment:
Recomendo colocar em computed
```suggestion
      v-if="hasItems"
```